modules/networking/main.tf

```diff
@@ -32,6 +32,7 @@ resource "google_service_networking_connection" "default" {
   network                 = google_compute_network.vpc.id
   service                 = "servicenetworking.googleapis.com"
   reserved_peering_ranges = [google_compute_global_address.private_ip_address.name]
+  deletion_policy         = "ABANDON"
```

Comment:
What happens to the abandoned resources, are they cleaned up eventually?

Reply:
Discussed this with @mwhooker and we are going to merge this and observe what happens to the abandoned resources

Reply:
Without clear documentation, we will have to check if anything's dangling after we merge